src/python.rs

```diff
@@ -805,4 +803,24 @@ mod tests {
         assert!(PythonVersionInfo::from_str("3.5.2a1+") < (3, 6));
         assert!(PythonVersionInfo::from_str("3.5.2a1+") > (3, 4));
     }
+
+    #[test]
+    fn test_acquire_gil() {
```

Comment:
Looks like `PyGILState_Check` requires the unlimited api. (See #1787)

```suggestion
    #[test]
    #[cfg(not(Py_LIMITED_API))]
    fn test_acquire_gil() {
```